runtime/System/Console/Hawk/Runtime.hs

```diff
@@ -74,17 +85,23 @@ splitAtSeparator (Delimiter d) = Search.split d
 
 
 outputRows :: Rows a => OutputSpec -> a -> IO ()
-outputRows (OutputSpec _ spec) x = ignoringBrokenPipe $ do
+outputRows (OutputSpec out spec) x = ignoringBrokenPipe $ do
     let s = join' (toRows x)
-    B.putStr s
-    hFlush stdout
+    case out of
+        UseStdout           -> do B.putStr s; hFlush stdout
+        OutputFile f ""     -> do B.writeFile f s; hFlush stdout
```

Comment:
Why flush stdout if we write to a file, not stdout?

Reply:
I suppose we are not using file handles, so there's no need to flush?

Reply:
Indeed.